docs/how-to/unified_memory.rst

```diff
@@ -0,0 +1,595 @@
+.. meta::
+  :description: This chapter describes introduces Unified Memory (UM) and shows
+                how to use it in AMD HIP.
+  :keywords: AMD, ROCm, HIP, CUDA, unified memory, unified, memory, UM, APU
+
+*******************************************************************************
+Unified Memory
+*******************************************************************************
+
+Concept
+=======
+
+In a conventional architectures, both CPUs a GPUs have a dedicated memory,
+the RAM and VRAM, respectively. Inherent in this architectural design is the
+need for continuous memory copying to allow the processors to access the
+appropriate data. This reduces the maximum available memory capacity and
+bandwidth. One way to avoid these effects is to use Heterogeneous System
+Architectures (HSA) such as Unified Memory.
+
+The Unified Memory (UM) can be understood as a single memory address space that
+is accessible from any processor within a system. This setup enables
+applications to allocate data that can be read or written by code running on
+either CPUs or GPUs. The Unified memory model is shown in the figure below.
+
+.. figure:: ../data/unified_memory/um.svg
+
+AMD Accelerated Processing Unit (APU) is a typical example of a Unified Memory
```

Comment:
Recommended

AMD Accelerated Processing Unit (APU)  is a typical example of a Unified Memory Architecture. On a single die, a central processing unit (CPU) is combined with an integrated graphics processing unit (iGPU), and both have access to a high-bandwidth memory (HBM) module named Unified Memory. The CPU enables high-performance, low-latency operations, while the GPU is optimized for high throughput (data processed by unit time).

Reply:
Updated.